### ISSUE TITLE
Publishing

### ISSUE THREAD

**Issue by trevor-scheer** [apollo-server-integrations/apollo-server-integration-google-cloud-functions#12]
Hey @ernestoresende, are you ready to publish this package yet? I've tried inviting you to the npm org, but the invite has expired twice now (I just renewed it once more). Let me know if you need anything from me!

**Comment by ernestoresende** [comment]
Yeah, ready to publish. Sorry for the delay, I think I sent an email re-requesting an invite to the org over the weekend, but I'm not sure it went through.

In any case, already went trough improving some documentation for the `/example` directory, and I think it is in a good point to get published at the very least on an alpha release. Before the first major release, I would like some help in writing the integration test (since it seems to be a default for other integrations), but I'll open another issue specifically for this.

**Comment by trevor-scheer** [comment]
Great! Looks like you're in. You should be able to generate an NPM token from your account (or `npm login` via CLI) and publish when you're ready.

Alpha sounds great, and setting up integration testing before major also sounds like a good plan.

The template this repo is based on uses changesets for publishing, you're welcome to rip it out or use it. There's a quick rundown on how to use it in the template's README: https://github.com/apollographql/typescript-repo-template#changesets, but if you want to publish an alpha you'll need to use `pre` mode (which you'll need to read up a bit about in their docs).

Bug me here if you run in to any trouble, then we can close this out once you've got version published!

**Comment by ernestoresende** [comment]
It's live! I ended up switching from `changeset` to `release-please` based on the configuration used at the [H3 Integration](https://github.com/apollo-server-integrations/apollo-server-integration-h3/tree/main). It made a little bit more sense for me from a publishing workflow standpoint, and the auto-generation plays nice with semantic commits.
